languages.ts

```diff
@@ -0,0 +1,12 @@
+const languages = [
+  { id: "en", title: "English", isDefault: true },
```

Comment:
What is the difference between `isDefault` and `base`?

Reply:
None, I'll remove, but all this will also be removed once we get the languages from Variant Norge and Variant Sverige